ddtrace/contrib/django/patch.py

```diff
@@ -822,7 +822,11 @@ def _(m):
 
 
 def wrap_wsgi_environ(wrapped, _instance, args, kwargs):
-    return core.dispatch_with_results("django.wsgi_environ", (wrapped, _instance, args, kwargs)).wrapped_result.value
+    result = core.dispatch_with_results("django.wsgi_environ", (wrapped, _instance, args, kwargs)).wrapped_result
+    if result:
+        return result.value
+    else:
+        return wrapped(*args, **kwargs)
```

Comment:
Is it possible that we're calling `wrapped` twice here?

Reply:
I was about to say no, but yes, it's possible if wrapped is raising an exception. Back to draft.

Reply:
Better management of corner case done.